Formula/steem.rb

```diff
@@ -0,0 +1,33 @@
+class Steem < Formula
+  desc "Proof of Work blockchain with an unproven consensus algorithm."
+  homepage "https://steemit.com/"
+  url "https://github.com/steemit/steem.git",
```

Comment:
Probably due to the submodule: https://github.com/steemit/steem/blob/master/.gitmodules

which unfortunately isn't pegged to a particular revision, so ideally it would be installed as a resource instead. However, I believe @DomT4 has said there's no firm rule regarding submodules that just point to a master HEAD unconditionally being unacceptable.